### PR TITLE
gh-122613: Document PyLong_GetInfo() (part of Limited API)

### PR DESCRIPTION
**Pull request by skirpichev**
----
📚 Documentation preview 📚: https://cpython-previews--122280.org.readthedocs.build/

* Issue: gh-122613

### CODE REVIEW

File: Doc/c-api/long.rst [hunk 1]
```diff
@@ -514,6 +514,17 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
...
 
+.. c:function:: PyObject* PyLong_GetInfo(void)
+
+   On success, return a read only :term:`named tuple`, that holds
+   information about Python's internal representation of integers.
+   See :data:`sys.int_info` for description of individual fields.
+
+   On failure, return ``NULL`` with an exception set.
+
+   .. versionadded:: 3.2
```
Comment (encukou) [inline Doc/c-api/long.rst]:
```suggestion
   .. versionadded:: 3.1
```

It was added in commit bd7926478de92a2a0ef4440e1a9ae61b706a80d2.

Comment (skirpichev) [reply Doc/c-api/long.rst]: Indeed, I was looking on stable_abi.dat.  Perhaps, this could be corrected too in this pr?

Comment (skirpichev) [reply Doc/c-api/long.rst]: Done

**Comment by encukou** [comment]
The function itself was added in 3.1, before stable ABI itself was [introduced in 3.2](https://peps.python.org/pep-0384/).
`stable_abi.toml` records when functions were added to stable ABI; that was in 3.2.

**Comment by encukou** [comment]
Thank you, and sorry for the delay!

**Pull request merged**